cloud/google/node_init.sh

```diff
@@ -42,12 +42,17 @@ fi
 sudo bash ./${AGENT_INSTALL_SCRIPT}
 
 # Examine what kind of Docker image is on this machine to determine what to log.
-if docker images | grep ct-server; then
+DOCKER_REPOS_FILE=$(mktemp)
+docker images --format '{{ .Repository }}' > "${DOCKER_REPOS_FILE}"
```

Comment:
Only VMs running old images will have too old a version of Docker. Since node_init.sh is only run when a node is being initialized, we can be sure that it will have a new image and therefore a recent version of Docker. For example, running `docker -v` on a more recent VM prints `Docker version 1.11.2, build b9f10c9`.